main.go

```diff
@@ -168,9 +164,9 @@ func main() {
 		// Listen on our channel AND a timeout channel - which ever happens first.
 		select {
 		case <-c1:
-			fmt.Println("Simulation Finished Successfully")
+			log.Info("Simulation Finished Successfully")
```

Comment:
Can we add the simulation name in the message and a "Simulation {NAME} started"?

Reply:
Wouldn't it be reasonable to expect the simulation name to be the same as the folder name that is passed already?

Reply:
Oh true this is single local simulation, my bad!
It would be useful to log folder name for the http handler